src/main/java/uk/gov/ida/matchingserviceadapter/domain/EidasToVerifyLoaTransformer.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.ida.matchingserviceadapter.domain;
+
+import uk.gov.ida.matchingserviceadapter.rest.matchingservice.LevelOfAssuranceDto;
+import uk.gov.ida.saml.core.domain.AuthnContext;
+import uk.gov.ida.saml.hub.domain.LevelOfAssurance;
+
+import java.util.function.Function;
+
+public class EidasToVerifyLoaTransformer implements Function<EidasLoa, LevelOfAssuranceDto> {
```

Comment:
Why are we making it a functional interface? Do we need to?

Reply:
Agreed!